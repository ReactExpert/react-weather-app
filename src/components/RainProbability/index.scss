
.rain {
  align-items: center;
  display: flex;
  justify-content: center;
  margin-left: -6px;
  opacity: 0;
  transform: translateY(-20px);
  transition: transform .3s cubic-bezier(.165, .84, .44, 1), opacity .3s ease-out;

  &.animate-in {
    opacity: 1;
    transform: translateY(0);
  }

  &__icon {
    height: 24px;
    width: 24px;
  }

  &__icon svg {
    width: 24px;
  }

  &__probability {
    font-size: 16px;
  }
}
