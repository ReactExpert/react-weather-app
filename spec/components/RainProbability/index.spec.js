import React from 'react';
import RainProbability from '../../../src/components/RainProbability';
import { enzymeConfig, mount } from '../../enzymeConfig';

enzymeConfig();

describe('RainProbability', () => {
  it('matches snapshot', () => {
    const component = mount(<RainProbability probability={20}/>);

    expect(component).toMatchSnapshot();
  });
});
