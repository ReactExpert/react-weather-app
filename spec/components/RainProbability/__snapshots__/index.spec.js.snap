// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RainProbability matches snapshot 1`] = `
<RainProbability
  probability={20}
>
  <div
    className="rain"
  >
    <img
      alt="rain probability 20"
      className="rain__icon"
      height="24"
      src="rain-probality.svg"
      width="24"
    />
    <span
      className="rain__probability"
    >
      20%
    </span>
  </div>
</RainProbability>
`;
